src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -36,6 +36,7 @@
     public String youHaveBeenInvited = "%prefix% &7%inviter% has invited you to join their island.";
     public String alreadyInYourIsland = "%prefix% &7This player is already a member of your Island.";
     public String inviteRevoked = "%prefix% &7Island invite for %player% has been revoked.";
+    public String islandTeamFull = "%prefix% &7Island maximum member count reached";
```

Comment:
Sounds a little bit odd, mostly because there is no verb.

Maybe "The maximum amount of members has already been reached"?

Reply:
my bad english..

Reply:
Oh wait this sends on player tried join a full island too
May it should om another message what your things ?

Reply:
Yes you should

Reply:
It may only be sent to the player who can't join

Reply:
How about something like "This island is currently full" or "The island member count has been reached" or smthn